VAMobile/src/components/MessageAlert.test.tsx

```diff
@@ -21,22 +14,20 @@ context('MessageAlert', () => {
     savingDraft?: boolean
     errorList?: { [key: number]: string }
   }): void => {
-    component = render(<MessageAlert hasValidationError={hasValidationError} saveDraftAttempted={saveDraftAttempted} />)
-
-    testInstance = component.UNSAFE_root
+    render(<MessageAlert hasValidationError={hasValidationError} saveDraftAttempted={saveDraftAttempted} />)
   }
 
   it('displays save draft validation', async () => {
     initializeTestInstance({ hasValidationError: true, saveDraftAttempted: true })
-    expect(testInstance.findAllByType(AlertBox).length).toEqual(1)
-    expect(findByTypeWithText(testInstance, TextView, 'We need more information')).toBeTruthy()
-    expect(findByTypeWithSubstring(testInstance, TextView, 'To save this message, provide this information:')).toBeTruthy()
+    expect(screen.UNSAFE_getAllByType(AlertBox).length).toEqual(1)
```

Comment:
I believe these unsafe checks can be removed

Reply:
Updated